Automattic/wp-calypso
# Reader > Discover: remove featured posts panel

#### Changes proposed in this Pull Request

We are no longer publishing Discover features weekly and will publish more on an ad hoc basis moving forward. This means that the three most recent posts displayed in the “Highlights: What we’re reading this week” section at the top of wordpress.com/discover will almost always be old and outdated.

This PR removes the highlights section from http://calypso.localhost:3000/discover.

<img width="388" alt="screen-shot-2019-05-14-at-6 35 03-am" src="https://user-images.githubusercontent.com/17325/57712687-51f67880-7671-11e9-94d2-24477b0e7b75.png">

#### Testing instructions



Visit http://calypso.localhost:3000/discover and ensure that the highlights section is no longer shown. The Discover stream should be visually identical to other streams, like http://calypso.localhost:3000/read/feeds/93876561.
